question/bank/previewquestion/amd/src/preview.js

```diff
@@ -16,16 +16,89 @@
 /**
  * Javascript for preview.
  *
+ * @module     qbank_preview/preview
  * @copyright  2021 Catalyst IT Australia Pty Ltd
  * @author     Safat Shahin <safatshahin@catalyst-au.net>
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
+import $ from 'jquery';
 
+/**
+ * Set up the actions.
+ *
+ * @method init
+ * @param {bool} redirect Redirect.
+ */
 export const init = (redirect) => {
     if (!redirect) {
         let closeButton = document.getElementById('close-previewquestion-page');
         closeButton.onclick = () => {
             window.close();
         };
     }
+
+    let form = document.getElementById('responseform');
+    // Turning off browser autocomplete.
```

Comment:
pretty minor, but just to be safe, please add a validation before calling those functions, if (form)